.github/ISSUE_TEMPLATE/support-request.yml

```
@@ -32,3 +32,7 @@ body:
   attributes:
     label: Additional context
     description: Please attach here any material you may consider useful (pictures, screenshots, logs, etc.).
+- type: textarea
+  attributes:
+    label: How does it affect me?
```

```suggestion
    label: How does it affect you?
```

`You` is coherent with the rest of the text.